lab3/src/pages/dashboard/calendar/index.html

```diff
@@ -0,0 +1,123 @@
+            alt=""
+            class="sidebar-item__icon"
+          />
+        </div>
+        <a class="sidebar-item" data-section="home" href="/dashboard">
+          <img
+            src="/assets/icons/sidebar/home.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">ГЛАВНАЯ</div>
+        </a>
+        <div class="sidebar-item" data-section="classes">
+          <img
+            src="/assets/icons/sidebar/stat.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">ПРЕДМЕТЫ</div>
+        </div>
+        <div class="sidebar-item sidebar-item--selected" data-section="calendar">
+          <img
+            src="/assets/icons/sidebar/calendar.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">КАЛЕНДАРЬ</div>
+        </div>
+        <div class="sidebar-item" data-section="messages">
+          <img
+            src="/assets/icons/sidebar/message.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">СООБЩЕНИЯ</div>
+        </div>
+        <div class="sidebar-item" data-section="homework">
+          <img
+            src="/assets/icons/sidebar/book.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">ДОМАШНЯЯ РАБОТА</div>
+        </div>
+        <div class="sidebar-item" data-section="settings">
+          <img
+            src="/assets/icons/sidebar/gear.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">НАСТРОЙКИ</div>
+        </div>
+        <div class="sidebar-item" data-section="help">
+          <img
+            src="/assets/icons/sidebar/question.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text">ПОМОЩЬ</div>
+        </div>
+        <div class="flex-grow"></div>
+        <div class="sidebar-item">
+          <img
+            src="/assets/icons/sidebar/exit.svg"
+            alt=""
+            class="sidebar-item__icon"
+          />
+          <div class="sidebar-item__text"></div>
+        </div>
+      </div>
+
+      <div class="dashboard-header">
+        <div class="dashboard-header__block dashboard-header__search-wrapper">
+          <img
+            src="/assets/icons/search.svg"
+            alt=""
+            class="dashboard-header__icon"
+          />
+          ПОИСК
+        </div>
+        <div class="dashboard-header__block">
+          <img
+            src="/assets/icons/bell.svg"
+            alt=""
+            class="dashboard-header__icon"
+          />
+          <div class="dashboard-header__block">
+            <img
+              src="/assets/images/user.png"
+              alt=""
```

Comment:
You either fill the `alt` value, or fully omit this attribute.

Reply:
ok

Reply:
fixed